admin-dev/themes/new-theme/template/helpers/kpi/kpi.tpl

```diff
@@ -39,8 +39,10 @@
       </div>
     {/if}
     <span class="title">{$title|escape}</span>
-    <span cLass="subtitle">{$subtitle|escape}</span>
-    <span class="value">{$value|escape|replace:'&amp;':'&'}</span>
+    <div class="kpi-description">
+      <div cLass="subtitle">{$subtitle|escape}</div>
```

Comment:
```suggestion
      <div class="subtitle">{$subtitle|escape}</div>
```

Reply:
Nice catch !

Reply:
I cant merge the suggestion 😢 github tells me I have no permission to do this

Reply:
I think it was in legacy - super duper hard to see such things ;D